Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "finality-grandpa"
-version = "0.14.4"
+version = "0.14.5"
```

Comment:
I think this should be a major bump to "0.15.0" as people depending on `Encode/Decode` of these types will also need to bump their scale codec version.

Reply:
Cool, I think we can do all of that in this PR :)